### ISSUE TITLE
[feature request] Ability to run multiple instances of ElectronIM, each with their own tab groups

### ISSUE THREAD

**Issue by CocaineJohnsson** [manusa/electronim#363]
It would be nice to be able to define multiple windows in addition to tabs, each with their own tabs. Ideally with the ability to close/reopen them independently.

I could not figure out any way to do this as the software currently functions, might be possibly by abusing snap or docker instances but I do not wish to pursue that method.

**Comment by CocaineJohnsson** [comment]
It's much better than nothing, but it'd get a bit tedious past 2-3 instances (it would be much more _convenient_ to be able to do this via the GUI), I'll evaluate this solution for my use-case.

**Comment by manusa** [comment]
Besides the multiple windows, I assume that your expected behavior would be that each window shared an isolated context for its tabs (unless the tab is sandboxed). Right?

Could you also please elaborate on your use-case? (My assumption is that you might want this to have a different window for separate contexts -home, work, etc.-)

**Comment by CocaineJohnsson** [comment]
Yes, and that they can be closed independently of each other without bringing down electronIM as a whole
